exporter/prometheusremotewriteexporter/exporter.go

```diff
@@ -180,18 +182,27 @@ func (prwe *PRWExporter) PushMetrics(ctx context.Context, md pdata.Metrics) erro
 	}
 }
 
-func validateAndSanitizeExternalLabels(externalLabels map[string]string) (map[string]string, error) {
+func validateAndSanitizeExternalLabels(externalLabels map[string]string, sanitizeLabel bool) (map[string]string, error) {
```

Comment:
```suggestion
func validateAndSanitizeExternalLabels(cfg *Config) (map[string]string, error) {
```

The `Config` struct has both the external labels and the sanitization flag.  Passing it here means there is no need to change the signature again when the feature gate is removed.

Reply:
updated